pages/api/users/[userId].js

```diff
@@ -7,15 +7,25 @@ export default async function handler(request, response) {
 
   try {
     connectDb();
-
+    const getUser = await User.findById(userId).exec();
     const session = await getSession({ req: request });
 
     switch (request.method) {
       case "GET":
-        const getUser = await User.findById(userId).exec();
         response.status(200).json(getUser);
         break;
 
+      case "PATCH":
+        const updatedUser = await User.findByIdAndUpdate(
+          userId,
+          {
+            ...request.body,
+          },
```

Comment:
I think you can get away without copying the body here :)

```suggestion
          request.body,
```